.mergify.yml

```diff
@@ -0,0 +1,35 @@
+pull_request_rules:
+  - name: Label the approved PRs
+    conditions:
+      - "#approved-reviews-by>=1"
+    actions:
+      comment:
+        message: "Hello :wave: @{{author}}, The PR is eligible to merge. It will be automerged after 3 working days if not raised any issues."
```

Comment:
```suggestion
       message: "Hello :wave: @{{author}}, this OFEP is eligible to be merged. It will be automatically merged after 3 working days if no objections are raised."
```